web/styles/settings.css

```diff
@@ -602,8 +602,9 @@ input[type="checkbox"] {
 }
 
 #add-custom-profile-field-button {
-    float: right;
-    margin-top: 4px;
+    margin-top: auto;
+    margin-bottom: 12px;
+    padding: 4px 14px;
```

Comment:
We use height property instead of setting in the top and bottom paddings for the "Add default channel" button, but this seems better.

Also, would be good to share the screenshot for this button since its design has been changed.

Reply:
Since I put the "Add default channel" button is inside a flexbox with "align-items" that defaults to "stretch", the height property of that button is not needed right now, and it uses the padding set from the .button class. I can remove the height property from that button.

Reply:
It works fine for the two font-settings that we support in prod currently, but might break things at larger font-sizes where the search input height is still fixed. But should be fine for now.